guide/02-api-overview/release_notes_210.ipynb

```diff
@@ -0,0 +1,897 @@
+    "* [Advanced Machine Learning](/python/samples)\n",
+    "  * 2d Computer Vision\n",
+    "    * Object Detection\n",
+    "      * [Detecting Palm Trees using Deep Learning](/python/samples/detecting-palm-trees-using-deep-learning)\n",
+    "      * [Detecting Swimming Pools using Automated Deep Learning](/python/samples/detecting-swimming-pools-using-automated-machine-learning)\n",
+    "    * Feature Extraction\n",
+    "      * [Streams extraction using deep learning](/python/samples/streams-extraction-using-deeplearning)\n",
+    "      * [Extracting features and Land Use Land Cover using Panoptic Segmentation ](/python/samples/extracting0-features-and-land-use-land-cover-using-panoptic-segmentation)\n",
+    "      * [Coastline extraction using Landsat-8 multispectral imagery and band ratio technique](/python/samples/coastline-extraction-usa-landsat8-multispectral-imagery)\n",
+    "    * Object Classification\n",
+    "      * [Traffic Light Detection in Oriented Imagery Using ArcGIS Pretrained Model](/python/samples/traffic-light-detection-on-oriented-imagery)\n",
+    "    * Generative Models\n",
+    "      * [Generating Land Surface Temperature from multispectral imagery using Pix2Pix](/python/samples/generating-lst-from-multispectral-imagery-using-pix2pix)\n",
+    "      * [Generating cloud masks from satellite imagery - Part I ](/python/samples/cloud-detector-part1-cloudless-sentinel-&-unsupervised)\n",
+    "    * Other Imagery Use Cases\n",
+    "      * [Flood inundation mapping and monitoring using SAR data and deep learning](/python/samples/flood-inundation-mapping-using-sar-data-and-deep-learning)\n",
+    "      * [Cloud removal using pre-trained deep learning model and raster function](/python/samples/cloud-removal-using-pre-trained-deep-learning-model-and-raster-function)\n",
+    "      * [Property Valuation in King County USA using AutoML and Geoenriched data](/python/samples/house-price-prediction-using-automl)\n",
+    "* [Content Publishers](/python/samples)\n",
+    "  * [Publishing and Consuming OGC Services](/python/samples/publish-ogc-services)\n",
+    "\n",
+    "### Module Changes\n",
+    "\n",
+    "#### [`arcgis.gis`](/python/api-reference/arcgis.gis.toc.html)\n",
+    "* Adds module so methods can run without specifying backend: `Notebook Server` or `ArcGIS Online`:\n",
+    "  * [`arcgis.notebook`](/python/api-reference/arcgis.notebook.html)\n",
+    "    * [`create_snapshots()`](/python/api-reference/arcgis.notebook.html#create-snapshot)\n",
+    "    * [`execute_notebook()`](/python/api-reference/arcgis.notebook.html#execute-notebook)\n",
+    "    * [`list_runtimes()`](/python/api-reference/arcgis.notebook.html#list-runtimes)\n",
+    "    * [`list_snapshots()`](/python/api-reference/arcgis.notebook.html#list-snapshots)\n",
+    "* Adds Submodule:\n",
+    "  * [`ago.gis.agonb`](/python/api-reference/arcgis.gis.agonb.html)\n",
```

Comment:
Is this supposed to be arcgis.gis.agonb? Or is there a module named ago

Reply:
Nope. Fixed it.